hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -1191,7 +1195,12 @@ public boolean getRenameResilience() {
     return renameResilience;
   }
 
-  void setRenameResilience(boolean actualResilience) {
-    renameResilience = actualResilience;
+  public boolean isPaginatedDeleteEnabled() {
+    return isPaginatedDeleteEnabled;
+  }
+
+  @VisibleForTesting
```

Comment:
As discussed in the client comment, lets remove this public method.

Reply:
What is the issue with having a public method @VisibleForTesting here??
I see we have it for other configs as well.

Reply:
Problem is that its a setter. And, it can be invoked from outside abfs codebase : `fs.getAbfsStore(). getAbfsConfiguration().setIsPaginatedDeleteEnabled()`. Outside abfs should not be able to change the config. Other setters (except one or two) are non-public.

Requesting you to kindly remove it. The same behavior that you want to set / unset can be done by change in AbfsClient as suggested in: https://github.com/apache/hadoop/pull/6409#discussion_r1442539222

Reply:
Taken